nemo/collections/asr/parts/preprocessing/feature_loader.py

```diff
@@ -67,3 +78,16 @@ def process_segment(self, feature_segment):
             return torch.tensor(feature_segment, dtype=torch.float)
 
         return torch.tensor(feature_segment, dtype=torch.float)
+
+    def normalize_per_feature(self, x: torch.Tensor, return_mean_std: bool = False) -> torch.Tensor:
```

Comment:
Please change this to use this function: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/preprocessing/features.py#L59